jujugui/static/gui/src/app/components/revenue-statement/revenue-statement.js

```diff
@@ -0,0 +1,255 @@
+/* Copyright (C) 2018 Canonical Ltd. */
+'use strict';
+
+const React = require('react');
+
+const Spinner = require('../spinner/spinner');
+const BasicTable = require('../basic-table/basic-table');
+
+/**
+  React component used to display a list of the users revenue-statements in their profile.
+*/
+class RevenueStatement extends React.Component {
+  constructor() {
+    super();
+    this.xhrs = [];
+    this.state = {
+      data: [
+        {
+          id: 1
+        },
+        {
+          id: 2
+        }
+      ],
+      loading: false
+    };
+  }
+
+  render() {
+    let content;
+    if (this.state.loading) {
+      content = <Spinner />;
+    } else {
+      const rows = this.state.data.map(revenueStatement => {
+        return {
+          columns: [
+            {
+              content: 'equities-prod-lon-1',
+              columnSize: 4
+            },
+            {
+              content: '$0.028',
+              columnSize: 2
+            },
+            {
+              content: 'Hours',
+              columnSize: 2
+            },
+            {
+              content: '20%',
+              columnSize: 2
+            },
+            {
+              content: '$50.96',
+              columnSize: 2
+            }
+          ]
+        };
+      });
+      content = (
+        <div className="revenue-statement__inner">
```

Comment:
the div that is added as a wrapper in the return below doesn't have the `revenue-statement` class. If you're going to add that then you can disregard my comment above about the return.